packages/ferdi-deb/ferdi-deb.pacscript

```diff
@@ -2,7 +2,7 @@ name="ferdi-deb"
 pkgname="ferdi"
 version="5.6.3"
 url="https://github.com/getferdi/ferdi/releases/download/v${version}/ferdi_${version}_amd64.deb"
-breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb "
+breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb"
```

Comment:
Why was this modified here. Make reverse this change, and make a separate PR for this.

Reply:
The commit is "remove extra space at the end" as the line is exactly the same in the two packages, it was combined in one commit.

Reply:
Thank you for the conventionnal commits system link, I didn’t know about it

Reply:
You really want me to revert the commit, and create a separate commit for a space typo?

Reply:
Let's try to keep the PR on topic (the topic is about adding `google-earth-pro-stable-deb`). Off-topic changes (though maybe discussed or brought up in this PR), which aren't hard-depended on by this PR should be made in their separate PR. This helps to maintain a clean commit history, as many (or rather one) of the users of pacstall uses a RSS feed to filter out changes.

So it's best to keep the changes on topic and code systematically.

Reply:
Sorry I had not seen that the README.MD had changed and now includes stuff about pro-hooks and stuff.

Ok I will revert